### DISCUSSION TITLE
PAGFile.LoadAsync方法加载大文件OOM

### DISCUSSION THREAD

**Discussion by JDLLXY** [Tencent/libpag#1942] [General]
【版本信息】
PAG SDK 版本 4.3.33
【平台信息】
Android4.4 原生 
【日志】
![微信图片_20231130160935](https://github.com/Tencent/libpag/assets/17818943/97952d40-aefb-4561-91ed-f1c2b5fd4833)
![微信图片_20231130160926](https://github.com/Tencent/libpag/assets/17818943/2db23751-51e8-4261-af9b-03908d2102e8)

**Comment by JDLLXY** [comment]
使用PAG资源大小为32M，同样放在Assets目录中没有发生OOM

**Comment by domchen** [comment]
可以稳定复现的话构造一个demo上传上来，这样定位问题比较快。

**Comment by JDLLXY** [comment]
就是用官方demo运行的，只是文件换了。

**Comment by CodeJhF** [comment]
抱歉这条后来没有继续跟进：当时你回复用官方 demo 换掉文件即可复现，我们漏了这一步。

这条上报的 libpag 版本是 4.3.33（4.3 及更早版本已停止维护，当前仅维护 4.4 / 4.5），先关闭本条。
如果这个 32M 素材在最新版本（4.4 / 4.5）上仍然 OOM，请回复版本号 + 素材，我们会按内存问题重新排查。（stale-close）
